### PR TITLE
engine from magic support for texify

### PR DESCRIPTION
**Pull request by m0nhawk**
I have fixed the `texify` builder for MikTeX.

### CODE REVIEW

File: lib/builders/texify.js [hunk 1]
```diff
@@ -30,23 +30,38 @@ export default class TexifyBuilder extends Builder {
...
+      args.push(`--engine="${engineFromMagic}"`)
+    } else if (customEngine) {
+      args.push(`--engine="${customEngine}"`)
+    } else if (engine && engine === 'xelatex') {
+      args.push(`--engine=xetex`)
+    } else if (engine && engine === 'lualatex') {
+      args.push(`--engine=luatex`)
+    }
+
+    if (engine) {
```
Comment (thomasjo) [inline lib/builders/texify.js]: This entire `if (engine)` block can be removed since it's a duplication of the `if` chain above.

**Comment by thomasjo** [comment]
Thanks so much for taking care of this! Besides my one comment, this looks good. I'll merge as soon as you address the code duplication :bow:

**Comment by m0nhawk** [comment]
Oh, I just forget to remove redundant  code,

**Comment by thomasjo** [comment]
Thanks, you rock! :sparkles:

**Pull request merged**